### DISCUSSION TITLE
Offline file system browsing of Retype generated website

### DISCUSSION THREAD

**Discussion by ahmethakanbesel** [retypeapp/retype#515] [Q&A] [answered]
I don't want to deploy my docs online however when I use build files offline links don't work properly because there is no web server is running. Is there a option to give direct links to files?

![ss](https://user-images.githubusercontent.com/6422460/227305342-c10bc0db-e109-456a-9ef9-775421ef7abc.png)

**Comment by geoffreymcgill** [comment, marked as answer]
Add the following settings to your `retype.yml` project file to enable a offline browsing mode:

https://github.com/retypeapp/retype/issues/276#issuecomment-1458429826

```yml
search:
  preload: true

generator:
  directoryIndex:
    append: true
```

Please see [`preload`](https://retype.com/configuration/project/#preload) and [`append`](https://retype.com/configuration/project/#directoryindexappend) documentation.

Hope this helps.
